frontends/concrete-python/examples/key_value_database/key_value_database.ipynb

```diff
@@ -415,7 +410,7 @@
     "    #   keys = [[1, 0, 1, 0], [0, 1, 0, 1, 1]]\n",
     "    #   key = [1, 0, 1, 0]\n",
     "    #   equal_rows = [1, 0]\n",
-    "    equal_rows = (np.sum((keys - key) == 0, axis=1) == NUMBER_OF_KEY_CHUNKS)\n",
+    "    equal_rows = np.sum((keys - key) == 0, axis=1) == NUMBER_OF_KEY_CHUNKS\n",
```

Comment:
@umut-sahin my editor wants to change this file, do you want me to avoid it?

Reply:
no actually it was fixed on purpose by 95d48e1825a222c766cb35e8508512f56484ed70 so all good